instalacao.py

```diff
@@ -13,6 +13,10 @@ def instalar_categoria(numero_da_categoria, instalador, argumento_do_instalador)
 
 def instalacao_de_pacotes(escolha, categorias, instalador, argumento_do_instalador):
 	try:
+
+		with open('/etc/apt/sources.list.d/infosectools.list', 'r') as arquivo:
```

Comment:
Dá uma olhada nesse carinha aqui: https://www.freecodecamp.org/news/how-to-check-if-a-file-exists-in-python/

Acho que dessa forma, fica melhor a checagem de existência de arquivos do que tentar abrir ele